relay-general/Cargo.toml

```diff
@@ -12,7 +12,7 @@ publish = false
 bytecount = "0.6.0"
 chrono = { version = "0.4.7", features = ["serde"] }
 cookie = { version = "0.12.0", features = ["percent-encode"] }
-debugid = { version = "0.7.0", features = ["serde"] }
+debugid = { version = "0.7.1", path = "/Users/untitaker/projects/rust-debugid/", features = ["serde", "schemars"] }
```

Comment:
path is wrong

Reply:
yes it is running a fork of the dependency. I got some feedback from jan that we should avoid modifying the dependency though